emails/utils.py

```diff
@@ -341,3 +347,31 @@ def remove_message_from_s3(bucket, object_key):
             logger.error('s3_client_error_delete_email', extra=e.response['Error'])
         incr_if_enabled('message_not_removed_from_s3', 1)
     return False
+
+
+
+def count_tracker(html_content, tracker_domain, level='general'):
+    # html_content needs to be str for count()
+    tracker_count = html_content.count(tracker_domain)
+    study_logger.info(
```

Comment:
This logged a LOT, including when the tracker was not found in the email. I'm not sure this level of logging will be useful. What about including the non-zero counts in the `email_tracker_summary` message, and not logging for each tracker domain?